lib/Hakyll/Web/Feed.hs

```diff
@@ -120,6 +120,10 @@ renderFeed feedTpl itemTpl config itemContext items = do
             StringField s -> return s
             _             -> fail "Hakyll.Web.Feed.renderFeed: Internal error"
 
+    emailField = case feedAuthorEmail config of
+        Nothing    -> missingField
+        Just email -> constField "authorEmail" email
```

Comment:
Would you prefer a new helper

```
optField : String -> Maybe String -> Context a
```

used as `optField "authorEmail" (feedAuthorEmail config)`?